CHANGE_MANAGEMENT.md

```diff
@@ -0,0 +1,148 @@
+Although these processes are adapted from the [OSSWatch Meritocratic Model](http://oss-watch.ac.uk/resources/meritocraticgovernancemodel), this documentation is a formalization of existing processes involving relevant stakeholders.
+
+# Roles
+ There are four roles in the DQIC. 
+ 
+ ## Stakeholders
+Anyone can be a stakeholder; there are no special requirements; however, the most important stakeholders of this community consist of:
+- Members of government agencies that review and provide guidance as needed to support maturity of digital quality standards
+- Platform or technology owners that provide software to report on Digital Quality based on the Clinical Quality Language (CQL),
+- Payer and provider systems that use software to report on Digital Quality, and
+- Quality standard developers that maintain measures of quality in healthcare.
+
+Without these key stakeholders, the community would have limited value. Stakeholder participation is on a voluntary basis, but DQIC requests stakeholders to actively participate in the project and community as much as possible. Stakeholder contributions enable DQIC to ensure that they are satisfying the requirements of those stakeholders. Common stakeholder contributions include (but are not limited to):
+
+- Evangelism of the project (e.g. a link on a website and word-of-mouth awareness)
+- Informing developers of strengths and weaknesses from a new stakeholder perspective
+- Providing moral support (a ‘thank you’ goes a long way)
+- Stakeholders who continue to engage with the project and its community will often become more and more involved. Such stakeholders may find themselves becoming contributors, as described in the next section.
+
+## Contributors
+
+Contributors are community members who contribute in concrete ways to DQIC. Anyone can become a contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements and no selection process.
+
+In addition to their actions as stakeholders, contributors may also find themselves doing one or more of the following:
+
+- Contributing to the [Cinical Quality Language](https://github.com/cqframework/clinical_quality_language/tree/master) project
+- Supporting new users (existing users are often the best people to support new users)
+- Reporting bugs
+- Identifying requirements
+- Providing graphics and web design
+- Programming
+- Assisting with project infrastructure
+- Writing documentation
+- Fixing bugs
+- Adding features
+
+In addition, it is highly encouraged to be active Contributors engage with the project through the issue tracker and Zulip, or by writing or editing documentation. They submit changes to the project itself via pull requests, which will be considered for inclusion in the project by existing committers (see next section). Use of Zulip is the most appropriate place to ask for help when making that first contribution.
```

Comment:
```suggestion
In addition, Contributors are encouraged to engage with the project by [submitting issues](https://github.com/cqframework/cql-tests/issues) and [the CQL stream in the FHIR Zulip chat](https://chat.fhir.org/#narrow/stream/179220-cql), or by writing or editing documentation. Contributors can submit changes to the project itself via pull requests, which will be considered for inclusion in the project by existing committers (see next section). Use of Zulip is the most appropriate place to ask for help when making that first contribution.
```

Reply:
Much cleaner and like the links. Thanks!